### ISSUE TITLE
Multi currency support

### ISSUE THREAD

**Issue by donutloop** [chrisallenlane/cdash#5]
Hi there,
Currently, this tool views all prices in USD. Could we add support for other currency like EUR, what do you think about that idea?  

I only see one tiny issue. How do we want to get the different currency symbols (€,$,...), any idea? Maybe we replace the symbol notation with a 3 letter notation (EUR, USD,...).

Best regards,
Donutloop

**Comment by chrisallenlane** [comment]
@donutloop, +1. This was something I planned to implement when I had evidence that somebody other than me was using this thing :)

I've added a `--base` flag in `v1.2.0` that implements this functionality:
https://github.com/chrisallenlane/cdash/releases/tag/v1.2.0

There's still one nit to pick for another day: currently, `,` is always used as the thousands separator, and `.` is always used as the radix. I suspect that's incorrect for many of these added currencies. I'll sort that out some other time.

Closing as resolved. Thanks!
